docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+
+-------------------------------------------------------
+READMEs on READMEs (and other README-related resources)
+-------------------------------------------------------
+
+READMEs are a really important starting point for a code project, often the first thing people see about it. Fortunately, there are tons of great resources to help you write quality READMes. Check these out:
+
+* `Art of README <https://github.com/noffle/art-of-readme>`_
+* `Standard README <https://github.com/RichardLitt/standard-readme/>`_
+* `Write the Readable README <https://www.writethedocs.org/videos/na/2016/write-the-readable-readme-daniel-beck/>`_ (Daniel Beck @ Write the Docs NA 2016), and Daniel's `README checklist <https://github.com/ddbeck/readme-checklist>`_
+* `Hi, my name is README! <http://www.writethedocs.org/videos/eu/2017/hi-my-name-is-readme-raphael-pierzina/>`_ (Raphael Pierzina @ Write the Docs EU 2017)
+* `Ten Steps to a Better README <https://www.youtube.com/watch?v=PC05prd2usY>`_ (Mike Jang @ Ignite OSCON 2015)
+* `Developer Experience: GitHub READMEs <https://betta.io/blog/2017/02/07/developer-experience-github-readmes/>`_ (Cristlano Betta's blog post)
+* `READMEs and doc-driven development <https://www.writethedocs.org/blog/newsletter-august-2017/#readmes-and-doc-driven-development>`_ (August 2017  WTD newsletter)
+
+------------------------------------------
+For those who might consider freelancing …
+------------------------------------------
+
+… a generous Slack contributor provided a wealth of information and advice about what to look for and what to expect in the land of freelance work.
+
+We were planning to write this up for the newsletter, but very kindly the contributor did it for us! So you can read the write-up here: `Real Talk: Freelance/Contract Writing <https://dacharycarey.com/2019/06/12/real-talk-freelance-contract-writing/>`__
+
+A few other people also weighed in with some comments. Some don't work with a 1099 but set up their own corporations (S-Corp or LLC), and then work as their own employees. This approach can provide tax benefits for US workers, and can sometimes make you more appealing to a larger employer. People also suggested that professional support, in the form of a lawyer or accountant, is important if you're planning to work freelance long-term.
+
+And if you're going the independent route, don't forget to consider staffing agencies. They get a cut of whatever the client pays, but they sometimes provide benefits depending on the contract, and can ease the paperwork burden.
```

Comment:
```suggestion
And if you're going the independent route, don't forget to consider staffing agencies. They get a cut of whatever the client pays, but they sometimes provide benefits and ease the paperwork burden.
```

Reply:
That felt super wishy washy, but feel free to ignore.

Reply:
much nicer, ty